package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@govtechsg/dnsprove": "^2.0.5",
     "@govtechsg/oa-encryption": "^1.3.0",
-    "@govtechsg/oa-verify": "^2.1.0",
+    "@govtechsg/oa-verify": "^3.0.3",
```

Comment:
there is nothing to prevent merging while types are wrong in the project. I suggest to add a step to check types and run it in the CI

Reply:
![image](https://user-images.githubusercontent.com/503185/75840027-45602e00-5e05-11ea-8cad-092a11a58a43.png)

All this files have TS errors